app/uk/gov/hmrc/estates/connectors/DesConnector.scala

```diff
@@ -69,7 +71,7 @@ class DesConnector @Inject()(http: HttpClient, config: AppConfig) {
 
     implicit val hc: HeaderCarrier = HeaderCarrier(extraHeaders = desHeaders(correlationId))
 
-    Logger.info(s"[DesConnector] matching estate for correlationId: $correlationId")
+    logger.info(s"[checkExistingEstate] matching estate for correlationId: $correlationId")
```

Comment:
Do we want session here? (since you have a header carrier)

Reply:
Adam pointed out yesterday in one of the other services that where we are creating a new HeaderCarrier the session won't exist in it so it would always show "No Session ID available".

Reply:
Oh right. It's a new one. Yes, ignore me.